app/javascript/packs/stake_accounts/index_template.vue

```diff
@@ -250,6 +250,8 @@
   import jpoolImage from 'jpool.png'
   import daopoolImage from 'daopool.png'
 
+  import _ from 'lodash'
```

Comment:
Can't we import only debounce function ? 
Something like : 
```suggestion
  import { debounce } from 'lodash'
```
or 
```suggestion
import debounce from 'lodash.debounce';
```